defold-spine/editor/src/spineext.clj

```diff
@@ -767,13 +767,15 @@
 
 ;;//////////////////////////////////////////////////////////////////////////////////////////////
 
-(g/defnk produce-model-pb [spine-scene-resource default-animation skin material-resource blend-mode create-go-bones]
+(g/defnk produce-model-pb [spine-scene-resource default-animation skin material-resource blend-mode create-go-bones playback-rate offset]
   {:spine-scene (resource/resource->proj-path spine-scene-resource)
    :default-animation default-animation
    :skin skin
    :material (resource/resource->proj-path material-resource)
    :blend-mode blend-mode
-   :create-go-bones create-go-bones})
+   :create-go-bones create-go-bones
+   :playback-rate playback-rate
+   :offset offset})
```

Comment:
Always adding the `:playback-rate` and `:offset` here will result in these properties always being written to the project files. Typically when we add new fields, we make sure to not write them unless they deviate from the defaults so that you don't end up with modified files immediately after loading the project.

This is especially desirable when making changes to `:component` types, as they can be embedded inside game objects and collections that will also have unsaved changes from the extra properties.

You can use a `cond->` expression to `assoc` in the key-value pairs only when they deviate from the results like this:

```clj
(cond-> {:spine-scene (resource/resource->proj-path spine-scene-resource)
         :default-animation default-animation
         :skin skin
         :material (resource/resource->proj-path material-resource)
         :blend-mode blend-mode
         :create-go-bones create-go-bones}

        (not= 1.0 playback-rate)
        (assoc :playback-rate playback-rate)

        (not= 0.0 offset)
        (assoc :offset offset))
```

Reply:
fixed, thank you